source/salesposter/url-parameters.rst

```diff
@@ -0,0 +1,166 @@
+URL parameters
+=========================================
+Salesposter uses URL query string to control presentation and business logic.
+
+Query string parameters are:
+
```

Comment:
We should probably include a description of the "default" location of the sales poster, and a simple example (use the experimentation tenant as the tenantname in the example).

Reply:
What do you mean by "default" location? 

By simple example, do you mean a sample query?

Reply:
default location is something like: `https://{tenantName}-s4.azurewebsites.net/salesposter`  in case they havnt configured a custom domain.

By simple example i mean something like a sample query for ordering a specific subscription plan with "swedbank|payex" as the only payment provider (i.e. two parameters i guess). And then use experimentation as the example tenant...

Reply:
Done